public/styles/components/_hover-expander.scss

```diff
@@ -0,0 +1,26 @@
+.hover-expander {
+  position: relative;
+}
+
+.hover-expander,
+.hover-expander__trigger {
+  display: flex;
+}
+
+.hover-expander__content {
+  position: absolute;
+  top: 100%;
+  right: 0;
+  background: #fff;
+  min-width: 200px;
```

Comment:
Should there be a max-width too?

Reply:
not really a need at this point if you ask me?